kickstart/kickstart.json

```diff
@@ -1,15 +1,16 @@
 {
     "variables": {
```

Comment:
would like to see a variable for applicationName too unless we are going to create one generic "Sample App"